sdk/keyvault/azure-security-keyvault-keys/vcpkg/Config.cmake.in

```diff
@@ -4,7 +4,7 @@
 @PACKAGE_INIT@
 
 include(CMakeFindDependencyMacro)
-find_dependency(azure-core-cpp)
+find_dependency(azure-core-cpp "1.1.0")
```

Comment:
```suggestion
find_dependency(azure-core-cpp "1.2.0")
```

Reply:
@vhvb1989, sure! Both Key Vaults libraries, correct, including the secrets lib?
Thanks for reviewing!

Reply:
(found your comment below, you answered to my question)